installing.md

```diff
@@ -0,0 +1,288 @@
+# Installing Pelias
+
+Mapzen offers the Mapzen Search service in hopes that as many people as possible will use it,
+but we also encourage people to set up their own Pelias instance. Whether it's to import their own data,
+make their own tweaks to Pelias code, or to help with Pelias development, its important that we
+document how this can be done. Similarly, while there are ways this process can be
+[automated](https://github.com/pelias/vagrant), these instructions are written as if the setup is
+manual, to illustrate all the moving pieces of Pelias.
+
+## Gather the Ingredients
+
+In general, Pelias will require:
+
+* A working [Elasticsearch](https://www.elastic.co/products/elasticsearch) 1.7 cluster. It can be on
+  a single machine or across several
+* [Node.js](https://nodejs.org/) 0.12 or newer (Node 4 or 5 is recommended)
+* Up to 100GB disk space to download and extract data
+* Lots of RAM. At least 2-4GB. A full North America OSM import just barely fits on a machine with 16GB RAM
```

Comment:
2-4 GB doesn't seem like lots! :) Maybe we can make a higher minimum and recommended level of 16? (also "fits on" kind of sounds like storage space?)